app/models/actor.rb

```diff
@@ -0,0 +1,3 @@
+class Actor < ApplicationRecord
+  belongs_to :movie
+end
```

Comment:
The association you have here isn't quite right. Because there is a many-to-many relationship with Actors and Movies, we're actually going to want to have a join table and a model for that join table.

```ruby
class Actor < ApplicationRecord
  has_many :movie_actors
  has_many :movies, through: :movie_actors
end
```

```ruby
class MovieActor < ApplicationRecord
  belongs_to :actor
  belongs_to :movie
end
```

```ruby
class Movie < ApplicationRecord
  belongs_to :studio
  has_many :movie_actors
  has_many :actors, through: :movie_actors
end
```

```ruby
class Studio < ApplicationRecord
  has_many :movies
end
```